integrations/terraform_cloud/main.py

```diff
@@ -0,0 +1,144 @@
+from enum import StrEnum
+from typing import Any
+from client import TerraformClient
+from port_ocean.context.ocean import ocean
+from loguru import logger
+from asyncio import gather
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+import asyncio
+
+
+class ObjectKind(StrEnum):
+    WORKSPACE = "workspace"
+    RUN = "run"
+    STATE_VERSION = "state-version"
+
+
+def init_terraform_client() -> TerraformClient:
+    """
+    Intialize Terraform Client
+    """
+    config = ocean.integration_config
+
+    terraform_client = TerraformClient(
+        config["terraform_host"],
+        config["terraform_token"],
+    )
+
+    return terraform_client
+
+
+## Enriches the state version with output
+async def enrich_state_versions_with_output_data(
+    http_client: TerraformClient, state_versions: list[dict[str, Any]]
+) -> list[dict[str, Any]]:
+    async with asyncio.BoundedSemaphore(5):
```

Comment:
Set it higher i dont want the integration to be slow because of this

Reply:
Alright, taking it up to 20.